Rocket.Chat/Controllers/Chat/ChatViewController.swift

```diff
@@ -377,16 +364,37 @@ final class ChatViewController: SLKTextViewController {
         client?.runCommand(command: command, params: params, roomId: subscription.rid, errored: alertAPIError)
     }
 
-    fileprivate func sendTextMessage(text: String) {
+    fileprivate func sendTextMessage(videochatUrl: String? = nil) {
         guard
             let subscription = subscription,
+            let messageText = videochatUrl ?? textView.text
```

Comment:
This is kind confusing. We na have another method called `sendVideoCallTextMessage` and this one prepares the `text` param to call `sendTextMessage`. What do you think?